frontend/src/css/app/Cell.css

```diff
@@ -293,8 +293,8 @@
 }
 
 :root {
-  /* In wide mode, this will overextend, but that's fine */
-  --gutter-width: calc(50vw - (var(--content-width) / 2));
+  /* Set a fixed gutter width to ensure the hover area is always wide enough */
+  --gutter-width: 50px;
```

Comment:
Does this make the gutter smaller when the window is super wide? Can we do min(50px, ...)

Reply:
Done🙂